input/fsh/valuesets/au-erequesting-coverage-self-pay.fsh

```diff
@@ -0,0 +1,16 @@
+ValueSet: AUeRequestingCoverageSelfPay
```

Comment:
Commenting here to get an inline comment. 
Please update Terminology page, leaving the profile cell unfilled - we can update when the profile is created.

Reply:
Added AU eRequesting CoverageType Self-pay

Reply:
Closing this conversation; will continue in the inline terminology.md file.